### DISCUSSION TITLE
`pc-windows-msvc` vs `windows-msvc-full`

### DISCUSSION THREAD

**Discussion by goyalyashpal** [nushell/nushell#11400] [Q&A] [answered]
what is `pc` & which one to pick?

i tried searching for pc on https://www.nushell.sh/book/installation.html but didn't find anything

from: https://github.com/nushell/nushell/releases/tag/0.88.1 filtered with `C-f`: ".msi"

* [nu-0.88.1-aarch64-pc-windows-msvc.msi](https://github.com/nushell/nushell/releases/download/0.88.1/nu-0.88.1-aarch64-pc-windows-msvc.msi)
* [nu-0.88.1-aarch64-windows-msvc-full.msi](https://github.com/nushell/nushell/releases/download/0.88.1/nu-0.88.1-aarch64-windows-msvc-full.msi)
* [nu-0.88.1-x86_64-pc-windows-msvc.msi](https://github.com/nushell/nushell/releases/download/0.88.1/nu-0.88.1-x86_64-pc-windows-msvc.msi) | 13.8 MB
* [nu-0.88.1-x86_64-windows-msvc-full.msi](https://github.com/nushell/nushell/releases/download/0.88.1/nu-0.88.1-x86_64-windows-msvc-full.msi) | 21.7 MB

**Comment by fdncred** [comment]
our old windows prebuilt binaries were called pc-windows-msvc, so we just kept that name. these included and still include nushell compiled with default features. windows-msvc-full includes nushell built with features extra and dataframe, so it's the full build with every command in our repository available. this is the one i use every day, but i compile from the main branch of the repo.

**Reply by goyalyashpal** [reply]
> so we just kept that name

oh lol :D

**Reply by goyalyashpal** [reply]
i would suggest removing the `pc-` as it just causes confusion being unsymmetric
rather, any of these symmetric one would be much better i think:


* ❌ `pc-windows-msvc` vs `windows-msvc-full` : asymmetric & cryptic
* ✔️ `windows-msvc-default` vs `windows-msvc-full`
* ✔️  `windows-msvc` vs `windows-msvc-full`

**Reply by fdncred** [reply, marked as answer]
While I generally agree with your point, I'm not up to changing anything with the binaries because winget is so fickle and easy to break. It's working right now and I'm not interested in introducing anything that would break it.

**Reply by goyalyashpal** [reply]
ahw, ohkay. fair point 👍

**Reply by goyalyashpal** [reply]
hehe, 

i have switched to the awesome `scoop` package manager last week. 
it's so awesome. i think at least for windows, i'll never have to look back 😄
